tests/mgbench/benchmark.py

```diff
@@ -201,17 +209,25 @@ def filter_benchmarks(generators, patterns):
     dataset.prepare(cache.cache_directory("datasets", dataset.NAME, dataset.get_variant()))
 
     # Prepare runners and import the dataset.
-    memgraph = runners.Memgraph(
+    vendor = runners.Memgraph(
         args.memgraph_binary,
         args.temporary_directory,
         not args.no_properties_on_edges,
         args.bolt_port,
     )
+    # vendor = runners.Neo4j(
+    #     args.memgraph_binary, 
+    #     args.temporary_directory, 
+    #     args.bolt_port, 
+    #     )
+ 
     client = runners.Client(args.client_binary, args.temporary_directory, args.bolt_port)
-    memgraph.start_preparation()
+    vendor.start_preparation()
+    # client.execute(queries=[
+    #     ("DROP INDEX ON:User(id)", {}),
+    # ])
```

Comment:
0 commented code please 😄

Reply:
When setting trigger on neo4j, if it is there you get an exepction, I uncommented this, but I will updated dataset with this line.